utils/routes/index.ts

```diff
@@ -6,5 +6,9 @@ export const routes = {
   welcome: "/welcome",
   closet: "closet",
 
+  profileSettings: "profile/ProfileSettings",
```

Comment:
user `lowercase` e.g. `profile-settings` for pages. User `PascalCase` for components only